nuget/uap10.0/MainPage.xaml.txt.pp

```diff
@@ -4,7 +4,7 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:forms="using:Xamarin.Forms.Platform.WinRT"
```

Comment:
nunit-vs-templayes should be updated in sync with this. (Or an issue filed, at least!)

Reply:
And with the updated versions for Xamarin.Forms, etc. 👍